kano_settings/system/advanced.py

```diff
@@ -430,7 +430,10 @@ def redirect_traffic_to_localhost():
 
 
 def launch_sentry_server(filename):
-    subprocess.Popen(["sentry -c {}".format(filename)], shell=True)
+    subprocess.Popen(
```

Comment:
Don't know the specific reason here, but I'm a bit wary personally of using run_cmd as it can be rather slow.